### ISSUE TITLE
🐛 [BUG] - TypeError unsupported operand type(s) for *=: 'NoneType' and 'int'

### ISSUE THREAD

**Issue by drikusroor** [Amsterdam-Music-Lab/MUSCLE#1037]
## Problem Description

The error message is telling us that there's an unsupported operand type for the *= operation. 
The issue is happening in the `song_sync_continuation_score` function in the file `result/score.py` on line `71`. There, the line `previous_result.score *= -1` is trying to multiply a None value with an integer. I'm not sure if the previous result's score should always be defined or that we should perhaps initialise it when it doesn't exist and we cannot multiply.

See also https://uva-aml.sentry.io/issues/5413868492/?project=4506353897570304&query=is%3Aunresolved&referrer=issue-stream&statsPeriod=7d&stream_index=3

#### Sentry

<img width="875" alt="image" src="https://github.com/Amsterdam-Music-Lab/MUSCLE/assets/8208970/777e2120-dfaf-479a-a9ec-74351650ebc4">

#### Session results on Acceptance

As it tries to get the previous result with the key `"recognize"`, we can see that result is not always there (whether that's intentional or not), so that probably results into `previous_result = result.session.get_previous_result(['recognize'])` resolve into `None`. And from that you can obviously not read the `score` property.

<img width="1312" alt="image" src="https://github.com/Amsterdam-Music-Lab/MUSCLE/assets/8208970/ba17ee2d-3706-4357-9bd4-c997c97573a8">

### Reproduction URL

http://illc-aml-mu-prd.science.uva.nl/server/result/score/

### Reproduction steps

```bash
n/a
```

### Logs

```bash
https://uva-aml.sentry.io/issues/5413868492/?project=4506353897570304&query=is%3Aunresolved&referrer=issue-stream&statsPeriod=7d&stream_index=3
```

**Comment by drikusroor** [comment]
This probably happens due to the race condition that is fixed in #1043, but the questions is: should we handle a scenario like this or should we always assume that previous score is defined?

**Comment by drikusroor** [comment]
Most likely fixed by #1043 - the issue doesn't seem to appear anymore after of 2024-05-31
